packages/dd-trace/src/appsec/iast/vulnerability-reporter.js

```diff
@@ -56,7 +57,11 @@ function sendVulnerabilities (vulnerabilities, rootSpan) {
         // TODO: Store this outside of the span and set the tag in the exporter.
         tags[IAST_JSON_TAG_KEY] = JSON.stringify(jsonToSend)
         tags[MANUAL_KEEP] = 'true'
+
         span.addTags(tags)
```

Comment:
why did you change it to sample before adding tags ? when in the SDK it's adding tags before for example

Reply:
it doesn't matter. Just to keep the span tags functionality together.